app/web/src/features/dashboard/MyEvents.tsx

```diff
@@ -57,18 +64,23 @@ export default function MyEvents() {
               classes.cardContainer,
               classes.upcomingEventContainer
             )}
+            fetchNext={isBelowSm ? fetchNextPage : undefined}
+            hasMore={hasNextPage}
+            isFetching={isFetching}
```

Comment:
Wow, I totally forgot I put this functionality in :D

Reply:
Indeed, made my job very easy 😉